### PR TITLE
Feature/e UI 3792 implement manage link logic

### PR DESCRIPTION
**Pull request by udaydenduluri33**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-3792

### Change description ###
Implementing manage link logic

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

File: api/workAllocation2/constants/actions.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
+import { Action } from "workAllocation2/interfaces/task";
```
Comment (rayliangatos) [inline api/workAllocation2/constants/actions.ts:1]: this should be 
import { Action } from '../interfaces/task';

File: api/workAllocation2/caseWorkerService.ts [hunk 2]
```diff
@@ -0,0 +1,80 @@
+import { AxiosResponse } from 'axios'
```
Comment (rayliangatos) [inline api/workAllocation2/caseWorkerService.ts:1]: need to add the semicolon back

File: api/workAllocation2/interfaces/task.ts [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+    auto_assigned: boolean
+    actions: TaskAction[]
+    execution_type: string
+    assignee: string
+    dueDate: string
+    taskName: string
+    caseName: string
+    caseCategory: string
+    assigneeName: string
+    name: string
```
Comment (rayliangatos) [inline api/workAllocation2/interfaces/task.ts:32]: need to add the semicolon back

File: api/workAllocation2/interfaces/taskSearchParameter.ts [hunk 4]
```diff
@@ -0,0 +1,24 @@
+export interface TaskSearchParameters {
```
Comment (rayliangatos) [inline api/workAllocation2/interfaces/taskSearchParameter.ts:1]: need to add the semicolon back

File: api/workAllocation2/routes.ts [hunk 5]
```diff
@@ -0,0 +1,36 @@
+import { Router } from 'express'
+
+import {
+    getAllCaseWorkers,
+    getAllCaseWorkersForLocation,
+    getCaseWorkersForLocationAndService,
+    getCaseWorkersForService,
+    getTask,
+    postTaskAction,
+    postTaskSearchForCompletable,
```
Comment (rayliangatos) [inline api/workAllocation2/routes.ts:10]: need to add the semicolon back on whole file

File: api/workAllocation2/taskService.ts [hunk 6]
```diff
@@ -0,0 +1,31 @@
+import { AxiosResponse } from 'axios'
```
Comment (rayliangatos) [inline api/workAllocation2/taskService.ts:1]: semicolon

**Review by rayliangatos — APPROVED** [review]

**Review by leepavlouhmcts — APPROVED** [review]

**Pull request merged**